Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM bitwalker/alpine-erlang
```

Comment:
@webdeb I like it! Mind if I add it to this repo as an alternative to having the VM preinstalled in the container?